source/transactions/transactions.rst

```diff
@@ -580,9 +573,19 @@ Constructing the first command within a transaction
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 When constructing the first command within a transaction, drivers MUST
-add the ``lsid``, ``txnNumber``, ``readConcern``, ``startTransaction`` and
-``autocommit`` fields. This is an example of an insert command that
-begins a server transaction:
+add the ``lsid``, ``txnNumber``, ``startTransaction``, and ``autocommit``
+fields.
+
+Drivers MUST include the transaction's readConcern in the first command in a
+transaction if and only if the readConcern is supplied and not the default. The
```

Comment:
Can this say "not the server's default" so it's clear what "default" we're referencing?